src/containers/DemoComponents/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Stepper from '../../components/Stepper';
+import { DoublyLinkedList, Node } from '../../dataStructure/DoublyLinkedList';
+
+const DemoComponents: React.FC = () => {
+  const list = new DoublyLinkedList<string>();
```

Comment:
I feel like the users of this Stepper shouldn't be "almosst obligated" to use this new set to make the component work. The component should provide this functionality out of the box with a regular array provided.